index.js

```diff
@@ -82,6 +108,9 @@ function getPreviousTeamMember() {
 function getCurrentTeamMember() {
   return TEAM[index % TEAM.length];
 }
+function listCurrentTeamMember() {
```

Comment:
Is this function used? I don't see a reference to it. If not, it can be removed.

Reply:
You know what. I put it in there to use it, but never did.  It can be taken out

Reply:
Cool, please rip it out then, no need to keep around dead code.